frontend/.gitignore

```
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

gitbase-web added `.eslintcache`

```suggestion
yarn-error.log*

.eslintcache
```

---

not gitbase-web. Old version of `create-react-app`. New version doesn't do it. Most probably it's not needed anymore with newer eslint.